lib/loadObj.js

```diff
@@ -213,11 +213,11 @@ function loadObj(objPath, options) {
     }
   }
 
-  function createVertex(p, u, n) {
+  function createVertex(p, u, n, ignoreBadNormals = false) {
     // Positions
     if (defined(p) && globalPositions.length > 0) {
       if (p * 3 >= globalPositions.length) {
-        throw new RuntimeError(`Position index ${p} is out of bounds`);
+        //throw new RuntimeError(`Position index ${p} is out of bounds`);
```

Comment:
Should this also be wrapped in `if (!ignoreBadNormals) {` rather than commented out?

Reply:
sorry yes, that was the point of it all